northstar/src/runtime/mount.rs

```diff
@@ -67,8 +71,17 @@ impl std::fmt::Debug for MountControl {
 
 impl MountControl {
     pub(super) async fn new() -> Result<MountControl, Error> {
+        debug!("Opening loop control");
         let lc = LoopControl::open().map_err(Error::LoopDevice)?;
+        debug!("Opening device mapper control");
         let dm = devicemapper::DM::new().map_err(Error::DeviceMapper)?;
+
+        let dm_version = dm
+            .version()
+            .map_err(Error::DeviceMapper)
+            .map(Version::from)?;
```

Comment:
Why not make `dm.version()` directly return a `Version` type? this seems to be the only place where it is called. Or even simpler, if it is used just for logging, then just debug-print it?